build.open-xml-sdk.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off
+
+REM Restore NuGet packages
+.nuget\NuGet.exe restore Open-XML-SDK.sln
+
+REM Build release
+set FrameworkVersion=v4.0.30319
+set FrameworkDir=%SystemRoot%\Microsoft.NET\Framework
+
+if exist "%SystemRoot%\Microsoft.NET\Framework64" (
+  set FrameworkDir=%SystemRoot%\Microsoft.NET\Framework64
+)
+
+"%FrameworkDir%\%FrameworkVersion%\msbuild.exe" /p:Configuration=Release Open-XML-SDK.sln
```

Comment:
By using the full path for msbuild.exe, you can run the script from normal command prompts.